### ISSUE TITLE
Tensorflow BERT LAMB optimizer: why learning rate uses 0.5 polynomial decay

### ISSUE THREAD

**Issue by LiweiPeng** [NVIDIA/DeepLearningExamples#214]
The Tensorflow BERT code supports LAMB optimizer. From https://github.com/NVIDIA/DeepLearningExamples/blob/master/TensorFlow/LanguageModeling/BERT/optimization.py, the learning rate for LAMB uses power=0.5 for polynomial decay. The original LAMB paper https://arxiv.org/abs/1904.00962 uses power=1.0 for polynomial decay. 

Can you share data to show power=0.5 is better than power=1.0? Thanks.

**Comment by sharathts** [comment]
The power is a tune-able parameter, and at the time we spoke with the original authors, they had mentioned trying 0.5 with BERT. Version 3 generalizes the parameters to support other models and could potentially explain why it uses 1.0. We have not extensively compared different values with BERT.

If you don't mind sharing, please post your findings with other values. It may be helpful for other users here.

**Comment by LiweiPeng** [comment]
Thanks @sharathts . I only used power=1.0 and didn't compare different values of power. If I have a chance to compare them in the future, I'll share them here.
